chatgpt.js

```diff
@@ -1515,20 +1516,19 @@ const chatgpt = { // eslint-disable-line no-redeclare
     reviewCode() { chatgpt.code.review(); },
     scrollToBottom() { try { chatgpt.getScrollBtn().click(); } catch (err) { console.error(err.message); }},
 
-    send(msg, method='') {
-        for (let i = 0; i < arguments.length; i++) if (typeof arguments[i] !== 'string')
-            return console.error(`Argument ${ i + 1 } must be a string!`);
+    async send(msg, method='') {
+        if (typeof msg !== 'string') return console.error('Message must be a string!');
         const textArea = chatgpt.getChatBox();
         if (!textArea) return console.error('Chatbar element not found!');
-        const msgP = document.createElement('p'); msgP.textContent = msg;
+        const msgP = document.createElement('p'); 
+        msgP.textContent = msg;
         textArea.replaceChild(msgP, textArea.querySelector('p'));
         textArea.dispatchEvent(new Event('input', { bubbles: true })); // enable send button
-        setTimeout(function delaySend() {
+        setTimeout(() => {
             const sendBtn = chatgpt.getSendButton();
-            if (!sendBtn?.hasAttribute('disabled')) { // send msg
-                method.toLowerCase() == 'click' || chatgpt.browser.isMobile() ? sendBtn.click()
-                    : textArea.dispatchEvent(new KeyboardEvent('keydown', { key: 'Enter', bubbles: true }));
-            } else setTimeout(delaySend, 222);
+            if (!sendBtn?.hasAttribute('disabled')) {
+                method.toLowerCase() === 'click' || chatgpt.browser.isMobile() ? sendBtn.click() : textArea.dispatchEvent(new KeyboardEvent('keydown', { key: 'Enter', bubbles: true }));
+            }
```

Comment:
This breaks the original intended behavior of truly wait for Send button to be enabled cuz now it only checks once, also the ternary line was much more readable originally as 2 lines vs. 1 really long one